Unable to Build With Method 3

[CMakeOutput.log](https://github.com/DanBloomberg/leptonica/files/5365613/CMakeOutput.log)
[CMakeError.log](https://github.com/DanBloomberg/leptonica/files/5365614/CMakeError.log)
[CMakeError.log](https://github.com/DanBloomberg/leptonica/files/5365615/CMakeError.log)
[CMakeOutput.log](https://github.com/DanBloomberg/leptonica/files/5365616/CMakeOutput.log)



I am trying to build the leptonica library for over 2 weeks now.  I have the following error but I am unable to decipher what the issue is.  Please help.  My goal is to get leptonica compiling so I can move on to use tessaract


Below is the output of cmake
[jimik@localhost build]$ cmake .. -DBUILD_PROG=1
CMake Error at CMakeLists.txt:12 (cmake_policy):
  Policy "CMP0054" is not known to this version of CMake.


-- The C compiler identification is GNU 4.8.5
-- The CXX compiler identification is GNU 4.8.5
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Performing Test GIF_GifFileType_UserData
-- Performing Test GIF_GifFileType_UserData - Success
-- Found GIF: /usr/lib64/libgif.so (found version "4") 
-- Found JPEG: /usr/lib64/libjpeg.so  
-- Found ZLIB: /usr/lib64/libz.so (found version "1.2.7") 
-- Found PNG: /usr/lib64/libpng.so (found version "1.5.13") 
-- Found TIFF: /usr/lib64/libtiff.so (found version "4.0.3") 
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.27.1") 
-- Setting build type to 'Release' as none was specified.
-- Looking for include file dlfcn.h
-- Looking for include file dlfcn.h - found
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file memory.h
-- Looking for include file memory.h - found
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file stdlib.h
-- Looking for include file stdlib.h - found
-- Looking for include file strings.h
-- Looking for include file strings.h - found
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Looking for include file sys/stat.h
-- Looking for include file sys/stat.h - found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for include file openjpeg-2.0/openjpeg.h
-- Looking for include file openjpeg-2.0/openjpeg.h - not found
-- Looking for include file openjpeg-2.1/openjpeg.h
-- Looking for include file openjpeg-2.1/openjpeg.h - not found
-- Looking for include file openjpeg-2.2/openjpeg.h
-- Looking for include file openjpeg-2.2/openjpeg.h - not found
-- Looking for include file openjpeg-2.3/openjpeg.h
-- Looking for include file openjpeg-2.3/openjpeg.h - found
-- Looking for fmemopen
-- Looking for fmemopen - found
-- Looking for fstatat
-- Looking for fstatat - found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Using unsigned short
-- Check if the system is big endian - little endian
-- 
-- General configuration for Leptonica 1.80.0
-- --------------------------------------------------------
-- Build type: Release
-- Compiler: GNU
-- C compiler options:  
-- Linker options:   
-- Install directory: /usr/local
-- 
-- Build with sw [SW_BUILD]: OFF
-- Build utility programs [BUILD_PROG]: 1
-- Used ZLIB library: /usr/lib64/libz.so
-- Used PNG library:  /usr/lib64/libpng.so;/usr/lib64/libz.so
-- Used JPEG library: /usr/lib64/libjpeg.so
-- Used JP2K library: /usr/lib64/libopenjp2.so
-- Used TIFF library: /usr/lib64/libtiff.so
-- Used GIF library:  /usr/lib64/libgif.so
-- Used WEBP library: /usr/lib64/libwebp.so;/usr/lib64/libwebpmux.so
-- --------------------------------------------------------
-- 
-- Configuring incomplete, errors occurred!
See also "/home/jimik/dev/leptonica-1.80.0/build/CMakeFiles/CMakeOutput.log".
See also "/home/jimik/dev/leptonica-1.80.0/build/CMakeFiles/CMakeError.log".

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@
 #
 ###############################################################################
 
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.1.3)
 cmake_policy(SET CMP0054 NEW)
 
 # In-source builds are disabled.

